surfactant/cmd/cli.py

```diff
@@ -1,10 +1,55 @@
+import hashlib
+import sys
+
 import click
+from loguru import logger
+
+from surfactant.plugin.manager import find_io_plugin, get_plugin_manager
+from surfactant.sbomtypes._sbom import SBOM
 
 
 @click.argument("sbom", type=click.File("r"), required=True)
+@click.option("--file", is_flag=False, help="File of the entry to find")
+@click.option("--sha256", is_flag=False, type=str, help="sha256 hash of the entry to find")
+@click.option("--uuid", is_flag=False, type=str, help="uuid of the entry to find")
+@click.option(
+    "--installPath",
+    is_flag=False,
+    type=str,
+    help="Matches all entries with a install path or partial install path match",
+)
+@click.option(
+    "--containerPath",
+    is_flag=False,
+    type=str,
+    help="Matches all entries with a container path or partial container path match",
+)
+@click.option(
+    "--output_format",
+    is_flag=False,
+    default="surfactant.output.cytrics_writer",
+    help="SBOM output format, options=surfactant.output.[cytrics|csv|spdx]_writer",
```

Comment:
The short names for the plugins should also work for specifying the output format (`cytrics|csv|spdx|cyclonedx`).